src/test/scala/it/pagopa/interop/purposeprocess/RiskAnalysisValidationSpec.scala

```diff
@@ -150,9 +271,51 @@ class RiskAnalysisValidationSpec extends AnyWordSpecLike {
 
     }
 
-    // Fields are all required for now
-//    "succeed if not required field is missing" in {}
+    "succeed only formal (fail on missing expected answer (answer tree is not complete))" in {
+      val riskAnalysis = RiskAnalysisForm(
+        version = "1.0",
+        answers = Map(
+          "purpose"                    -> List("purpose"),
+          "usesPersonalData"           -> List("NO"),
+          "usesThirdPartyPersonalData" -> List("YES"),
+          "usesConfidentialData"       -> Nil,
+          "securedDataAccess"          -> Nil
+        )
+      )
+
+      val result: ValidationResult[RiskAnalysisFormSeed] = RiskAnalysisValidation.validateOnlyFormal(riskAnalysis)
+
+      val expected = RiskAnalysisFormSeed(
+        version = riskAnalysis.version,
+        singleAnswers = Seq(
+          SingleAnswerSeed("purpose", "purpose".some),
+          SingleAnswerSeed("usesPersonalData", "NO".some),
+          SingleAnswerSeed("usesThirdPartyPersonalData", "YES".some)
+        ),
+        multiAnswers = Seq.empty
+      )
 
+      verifyValidationFormResult(result, expected)
+
+    }
+
```

Comment:
let's add also a schema-only test where keys are correct but values are unexpected (the test should fail)